packages/vue-query/src/__tests__/useQuery.test.ts

```diff
@@ -192,26 +187,26 @@ describe('useQuery', () => {
 
     const enabled = computed(() => !!data.value)
 
-    const { fetchStatus, status } = useQuery(
+    const dependentQueryFn = jest.fn()
```

Comment:
I believe this mock fn could invoke `simpleFetcher` so it could be actually resolved